include/persistent_data.hpp

```diff
@@ -27,6 +27,8 @@ class ConfigFile
   public:
     // todo(ed) should read this from a fixed location somewhere, not CWD
     static constexpr const char* filename = "bmcweb_persistent_data.json";
+    static constexpr const char* dumpFilename =
+        "bmcweb_current_session_persistent_data.json";
```

Comment:
The filename is misleading.  Perhaps: bmcweb_current_session_snapshot.json.
Maybe also include the BMC's current TOD in the file so we know when the snapshot was taken?

Reply:
yes this name bmcweb_current_session_snapshot.json makes sense

Reply:
> bmcweb_current_session_snapshot.json 

+1

Reply:
done